app/src/main/java/org/oppia/android/app/policies/PoliciesFragmentPresenter.kt

```diff
@@ -64,6 +66,8 @@ class PoliciesFragmentPresenter @Inject constructor(
       supportsLinks = true,
       supportsConceptCards = false
     )
+    binding.policyDescriptionTextView.makeAutoLinks(policyDescription.getUrls())
```

Comment:
Why doesn't LinkifyCompat work for this? It's not clear to me why the custom extension functions are needed here.

Reply:
@BenHenning 
After getting all the URLs from the text I highlight them one by one. If we need to add more then one URL's in  the code will work fine.

LinkifyCompat not working here I don't know why.